sdk/confidentialledger/arm-confidentialledger/README.md

```diff
@@ -44,36 +44,34 @@ npm install @azure/identity
 ```
 
 You will also need to **register a new AAD application and grant access to Azure ConfidentialLedger** by assigning the suitable role to your service principal (note: roles such as `"Owner"` will not grant the necessary permissions).
-Set the values of the client ID, tenant ID, and client secret of the AAD application as environment variables: `AZURE_CLIENT_ID`, `AZURE_TENANT_ID`, `AZURE_CLIENT_SECRET`.
 
 For more information about how to create an Azure AD Application check out [this guide](https://learn.microsoft.com/azure/active-directory/develop/howto-create-service-principal-portal).
 
 Using Node.js and Node-like environments, you can use the `DefaultAzureCredential` class to authenticate the client.
 
 ```ts snippet:ReadmeSampleCreateClient_Node
-import { AstroManagementClient } from "@azure/arm-confidentialledger";
+import { ConfidentialLedgerClient } from "@azure/arm-confidentialledger";
```

Comment:
why this is changed?

Reply:
may forgot to replace the client name when do snippets migration